src/Package.php

```diff
@@ -7,45 +7,36 @@
 
 namespace Automattic\WooCommerce\Blocks;
 
+use Automattic\WooCommerce\Blocks\Domain\Package as NewPackage;
+
 defined( 'ABSPATH' ) || exit;
 
 /**
  * Main package class.
+ *
+ * @deprecated $VID:$
  */
 class Package {
 
 	/**
-	 * Version.
-	 *
-	 * @var string
-	 */
-	const VERSION = '2.5.0-dev';
-
-	/**
-	 * Stores if init has ran yet.
+	 * For back compat this is provided. Ideally, you should use register your
```

Comment:
There seems to be an unnecessary word here:
```suggestion
	 * For back compat this is provided. Ideally, you should register your
```